### DISCUSSION TITLE
It would be nice if YimMenu can export in `%APPDATA%/YimMenu` all Usernames and their IPs from players in the current session.

### DISCUSSION THREAD

**Discussion by BUZZARDGTA** [YimMenu/YimMenu#2927] [Ideas]
So the reason I'm requesting this feature is that I'm currently working on a project called [GTA V Session Sniffer](https://github.com/Illegal-Services/GTA-V-Session-Sniffer), which displays all connected players in a game lobby.
I'd like to add a feature specifically for users of YimMenu on PC, allowing them to view usernames of players directly within my project.
Does that make sense?

I'm thinking to prevent the creation of a massive database, you could display only the players **currently** in the session in the JSON file.
So when someone leaves the session, they would be automatically removed from the JSON.
This approach would help avoid ending up with an excessively large file size.

**Comment by D-Borkan** [comment]
it is not recommended to do that for privacy

**Reply by BUZZARDGTA** [reply]
I agree with that too
